engine/phonotype.sc

```diff
@@ -363,6 +364,18 @@ PTParser {
 		^(end -> newNode);
 	}
 
+	parseCrow { |preTokens, tokens, ctx|
```

Comment:
These methods are for PRE-style commands; I don't think crow needs to be that. Just adding "CROW" in the table in line 189 should be sufficient.